package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "test": "jest",
     "test:coverage": "jest --coverage --forceExit",
-    "migrate": "migrate",
+    "migrate": "node ./lock.js",
```

Comment:
and what about the commands "migrate:up" and "migrate:down"?

Reply:
I think migrate:up and migrate:down will be manual commands, so is ok to don't be locked, what do you think?